examples/__testfixtures__/styled-components-to-ucl/unsupported.output.ts

```diff
@@ -0,0 +1,25 @@
+import { Box, FormControl, Input, Text } from '@rbilabs/universal-components';
+
+export const Scrollable = Box.withConfig({
+  overflowX: 'auto',
+});
+
+export const Fieldset = FormControl.withConfig({
+  padding: 0,
+})/* TODO: RN - This was a <input> tag. Verify its styled properly*/;
+
+export const Container = Box.withConfig({})/* TODO: RN - This was a <ul> tag. Verify its styled properly*/;
+
+export const Label = Text.withConfig({
+  // TODO: RN - unsupported CSS
+  // display: 'block',
+})/* TODO: RN - This was a <label> tag. This should be converted to a UL <FormControl.Label>*/;
+
+export const Group = Box.withConfig({
+  listStyle: 'none',
```

Comment:
i don't think this is a valid css property fwiw

Reply:
lol, Okay, I'll add it to the remove props